crates/ra_ide/src/goto_implementation.rs

```diff
@@ -219,6 +219,10 @@ impl T for &Foo {}
   #[derive(Copy)]
 //^^^^^^^^^^^^^^^
 struct Foo<|>;
+
+mod marker {
+    trait Copy {}
+}
```

Comment:
This generated impl used to be recorded as an inherent impl since `Copy` couldn't be found.